kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/engine/DefaultExpressionHandler.java

```diff
@@ -38,12 +35,14 @@ public class DefaultExpressionHandler implements ExpressionHandler {
   @Override
   public ExpressionEvaluator getEvaluator(
       StructType inputSchema, Expression expression, DataType outputType) {
-    return new DefaultExpressionEvaluator(inputSchema, expression, outputType);
+    return new io.delta.kernel.defaults.internal.expressions.DefaultExpressionEvaluator(
```

Comment:
This is required for the unidoc to work (otherwise javadoc compilation fails since internal classes are filtered from the javadoc source files)